libnmstate/schemas/operational-state.yaml

```diff
@@ -435,3 +419,23 @@ definitions:
                     type: boolean
                   state:
                     type: string
+
+  route:
+    type: object
+    properties:
+      state:
+        type: string
+        enum:
+          - absent
+      table-name:
+        type: string        # empty for unknown
```

Comment:
I would prefer `unknown` to be expressed by not specifying it in the config.
It is also not common to have comments in schemas and an empty string is self explanatory.

Reply:
IMHO the empty string is not self explanatory because it could mean both that the table has no name or that the name of the table is unknown. But why can the table name be unknown?

Reply:
`table-name` has been removed.